eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -7,6 +7,10 @@ stages:
     jobs:
     - template: ../jobs/archetype-sdk-client.yml
       parameters:
+        ${{ if eq(parameters['ServiceDirectory'], 'core') }}:
```

Comment:
Can you make ServicetoBuild a parameter instead? And set it to `*` in core and default it to ServiceDirectory everywhere else? I'd prefer not to special case core in the template directly.

Reply:
Ok, I can do that, but that'll will mean modifying all the `ci.yml` files which I assumed will not be such a good idea.

Reply:
You should only need to modify the core ci.yml file. The others will get defaulted in this template file.